.github/workflows/simplye.yml

```diff
@@ -0,0 +1,84 @@
+name: SimplyE Build, Test, & Distribute
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Reposistory
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
+      - name: Set Up Credentials
+        env:
+          NYPL_S3_ACCESS_KEY: ${{ secrets.NYPL_S3_ACCESS_KEY }}
+          NYPL_S3_SECRET_ACCESS_KEY: ${{ secrets.NYPL_S3_SECRET_ACCESS_KEY }}
+          MAVEN_CENTRAL_USERNAME: ${{ secrets.MAVEN_CENTRAL_USERNAME }}
+          MAVEN_CENTRAL_PASSWORD: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
+          MAVEN_CENTRAL_STAGING_PROFILE_ID: 'af061f5afba777'
+          MAVEN_CENTRAL_SIGNING_KEY_ID: 'Library Simplified'
+          NYPL_GITHUB_ACCESS_TOKEN: ${{ secrets.NYPL_GITHUB_ACCESS_TOKEN }}
+        run: .ci/ci-credentials.sh
+
+      - name: Build
+        env:
+          NYPL_S3_ACCESS_KEY:               ${{ secrets.NYPL_S3_ACCESS_KEY }}
+          NYPL_S3_SECRET_ACCESS_KEY:        ${{ secrets.NYPL_S3_SECRET_ACCESS_KEY }}
+          MAVEN_CENTRAL_USERNAME:           ${{ secrets.MAVEN_CENTRAL_USERNAME }}
+          MAVEN_CENTRAL_PASSWORD:           ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
+          MAVEN_CENTRAL_STAGING_PROFILE_ID: 'af061f5afba777'
+          MAVEN_CENTRAL_SIGNING_KEY_ID:     'Library Simplified'
+          NYPL_GITHUB_ACCESS_TOKEN:         ${{ secrets.NYPL_GITHUB_ACCESS_TOKEN }}
+        run: .ci/ci-build.sh normal
```

Comment:
As part of this PR I would update this ci-build.sh script in the CI module to not do all the things it's doing right now.

Currently its doing `assemble test verifySemanticVersioning` - it could probably just do `assembleRelease` which will cut the runtime down significantly

Reply:
Good idea!

Reply:
Oh also you can probably adjust the parameters for the Gradle invocation to be similar to the one for pull-requests - i.e. `-Dorg.gradle.parallel=true`